vwifi.c

```diff
@@ -137,6 +293,17 @@ static inline struct owl_vif *wdev_get_owl_vif(struct wireless_dev *wdev)
     return container_of(wdev, struct owl_vif, wdev);
 }
 
+static inline u32 vwifi_mac_to_32(const u8 *mac)
```

Comment:
Why did you pick up the hash function?

Reply:
We need a hash function that maps 48-bit MAC addresses into 32-bit integers, so that the integer can be treated as a key in `hash_add()` and `hash_for_each_possible()`.

This hash function is from [murmur hash](https://stackoverflow.com/questions/7666509/hash-function-for-string/57960443#57960443) and is once used in #PR11 as our hash function. I think there may be a better hash function but currently, this works fine.